src/dotnet/install.sh

```diff
@@ -147,9 +147,9 @@ apt_cache_package_and_version_soft_match() {
     . /etc/os-release
     local architecture="$(dpkg --print-architecture)"
 
+    apt-get update -y
     major_minor_version="$(echo "${requested_version}" | cut -d "." --field=1,2)"
-    package_name="$(apt-cache search "${partial_package_name}-[0-9].[0-9]" | awk -F" - " '{print $1}' | grep -m 1 "${partial_package_name}-${major_minor_version}")"
-    
+    package_name="$(apt-cache search "${partial_package_name}-[0-9].[0-9]" | awk -F" - " '{print $1}' | grep -v "source-built-artifacts" | grep -m 1 "${partial_package_name}-${major_minor_version}")"
```

Comment:
I feel like we could update the `${partial_package_name}-[0-9].[0-9]` regex to capture the expected name of the package better. Maybe something like this (untested):

```suggestion
    package_name="$(apt-cache search "${partial_package_name}-[0-9].[0-9]$" | awk -F" - " '{print $1}' | grep -m 1 "${partial_package_name}-${major_minor_version}")"
```

Reply:
Nice 💡 
Udpated